src/index.ts

```diff
@@ -1,4 +1,36 @@
-//Uncomment to see the choose pokemon page
-//const newTemplate = document.querySelector('#choose-page-template')!;
-//const app = document.querySelector('#app')!;
-//app.innerHTML = newTemplate.innerHTML
\ No newline at end of file
+import { showAModal, hideAModal } from "./modalPopUpFunctions";
+
+const renderStartingPage = (appDiv: HTMLDivElement) => {
+  const startingPage = document.querySelector(
+    "#starting-page-template"
+  ) as HTMLTemplateElement;
+  appDiv.innerHTML = startingPage.innerHTML;
+};
+const renderHowToPlayModal = (appDiv: HTMLDivElement) => {
+  const infoModal = document.querySelector(
+    "#info-modal-template"
+  ) as HTMLTemplateElement;
+  appDiv.innerHTML += infoModal.innerHTML;
+};
+const createHowToPlayButton = () => {
+  const howToPlayButton = document.querySelector(
+    "#how-to-play-button"
+  ) as HTMLUListElement;
+  const infoModal = document.querySelector(
+    "#info-modal-screen"
+  ) as HTMLDivElement;
+  howToPlayButton.addEventListener("click", () => {
+    showAModal(infoModal, "flex");
+  });
+  howToPlayButton.addEventListener("click", () => {
+    hideAModal(infoModal);
+  });
```

Comment:
same here with "A", also you can use shorthand:
```suggestion
  howToPlayButton.addEventListener("click", () => hideModal(infoModal));
```